internal/host/plugin/job_set_sync.go

```diff
@@ -451,7 +465,7 @@ func (r *SetSyncJob) upsertHosts(
 							Action: db.DoNothing(true),
 						}
 						if err := w.CreateItems(ctx, hi.ipsToAdd.toArray(), db.NewOplogMsgs(&oplogMsgs), db.WithOnConflict(onConflict)); err != nil {
-							return errors.Wrap(ctx, err, op, errors.WithMsg(fmt.Sprintf("adding ips %v for host %q", hi.ipsToAdd.toArray(), hi.h.GetPublicId())))
+							return errors.Wrap(ctx, err, op, errors.WithMsg(fmt.Sprintf("adding ips %v for host %q", hi.ipsToAdd.toArray(), ret.GetPublicId())))
```

Comment:
Just taking the chance to get your input on this error message.  I added it kind of incidentally on friday to help debug.  Prior there was no WithMsg attached.  I'm worried the msg is too verbose.

Reply:
I think it's probably okay. This won't go to a user; it'd go to observability. And the number of IPs should be pretty small.